src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/CaseloadService.kt

```diff
@@ -19,14 +21,21 @@ class CaseloadService(
   fun getPrisoner(nomisId: String) =
     getPrisonersByNumber(listOf(nomisId)).firstOrNull() ?: throw EntityNotFoundException(nomisId)
 
-  private fun PrisonerSearchPrisoner.toCaseloadItem() = CaseloadItem(
-    prisoner = this.toPrisoner(),
-    cvl = CvlFields(
-      licenceType = LicenceType.getLicenceType(this),
+  private fun PrisonerSearchPrisoner.toCaseloadItem(): CaseloadItem {
+    val sentenceDateHolder = this.toSentenceDateHolder()
```

Comment:
this will have to get more clever if we evolve this to cover more caseload functionality - it's using prisoner search to calculate hardstop date rather than any stored licence